File: TemplateNetCore-main/MercanciaSegura.DOM/Modelos/Cliente/RegimenFiscal.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MercanciaSegura.DOM.Modelos.Cliente
{
    [Table("Regimen_Fiscal")]
    public class RegimenFiscal
    {
        [Key]
        [Column("Regimen_Fiscal_ID")]
        public int RegimenFiscalId { get; set; }


        [Column("Descripcion")]
        [MaxLength(100)]
        public string? Descripcion { get; set; }

        [Column("Aplica_Moral")]
        public bool AplicaMoral { get; set; }

        [Column("Aplica_Fisica")]
        public bool AplicaFisica { get; set; }

        [Column("Codigo")]
        public int? Codigo { get; set; }
    }
}

File: TemplateNetCore-main/WebAdmin/Modelos/RegimenFiscal.vb
Namespace MercanciaSegura.DOM.Modelos
    Public Class RegimenFiscal
        Public Property RegimenFiscalId As Integer
        Public Property Descripcion As String
        Public Property AplicaMoral As Boolean
        Public Property AplicaFisica As Boolean
        Public Property Codigo As Integer?

        Public ReadOnly Property CodigoDescripcion As String
            Get
                Return $"{Codigo} - {Descripcion}"
            End Get
        End Property
    End Class

End Namespace
